installation/sql/postgresql/supports.sql

```diff
@@ -226,7 +226,7 @@ CREATE TABLE IF NOT EXISTS "#__fields_values" (
 "field_id" bigint DEFAULT 0 NOT NULL,
 "item_id" varchar(255) DEFAULT '' NOT NULL,
 "value" text
-);
+	);
```

Comment:
```suggestion
);
```
Please revert this change.